lib/bookstore-scrapper/prisma-client/prismaClient.mts

```diff
@@ -0,0 +1,4 @@
+// Instantiate new prisma client
+import pkg from '@prisma/client';
+const { PrismaClient } = pkg;
```

Comment:
I think that you should be able to do this in one line, eg. `import { PrismaClient } from "@prisma/client";`

Reply:
I tried to do this, but I get the error below while trying to run the module:
<img width="633" alt="Screen Shot 2022-01-17 at 3 38 28 PM" src="https://user-images.githubusercontent.com/40670885/149834531-86b8893b-6110-4742-8874-e125d26ab69b.png">

Reply:
Okay. That sounds odd, but if it's causing an issue then I'm okay with the other syntax